ibm/service/configurationaggregator/data_source_ibm_config_aggregator_configurations.go

```diff
@@ -77,75 +78,17 @@ func DataSourceIbmConfigAggregatorConfigurations() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"about": &schema.Schema{
-							Type:        schema.TypeList,
+							Type:        schema.TypeMap,
```

Comment:
If we go with Typemap then we have limitation from Terraform only can hold only string type. If you have combination of multiple types (string ,int, bool) then this will not be compatible.

Reply:
Can you think of declaring it string as json 
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/resource_key#credentials_json chk this code

Reply:
All fields of this attribute are of type string so there is no modification  required.